src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("start work");
```

Comment:
```suggestion
        System.out.println("Bulldozer start work");
```
аналогічно для інших методів, в кожної машини вони повинні відрізнятись, інакше ми просто дублюємо код

Reply:
Ок